spec/jobs/bulk_invite_spec.rb

```diff
@@ -83,20 +83,27 @@
 
     it 'can create staged users and prepulate user fields' do
       user_field = Fabricate(:user_field, name: "Location")
+      user_field_color = Fabricate(:user_field, field_type: "dropdown", name: "Color")
+      user_field_color.user_field_options.create!(value: "Red")
+      user_field_color.user_field_options.create!(value: "Green")
+      user_field_color.user_field_options.create!(value: "Blue")
+
       described_class.new.execute(
         current_user_id: admin.id,
         invites: [
           { email: 'test@discourse.org' }, # new user without user fields
-          { email: user.email, location: 'value 1' }, # existing user with user fields
-          { email: staged_user.email, location: 'value 2' }, # existing staged user with user fields
+          { email: user.email, location: 'value 1', color: 'blue' }, # existing user with user fields
+          { email: staged_user.email, location: 'value 2', color: 'redd' }, # existing staged user with user fields
```

Comment:
Is that typo wanted? 🤔

Reply:
Yes. I wanted to make sure that if a value is wrong, it will not be saved. Invalid values (that are not present in the options list) caused issues when showing the invite because the dropdown was set to a value that was not present in the list.